tests/internal/test_remoteconfig.py

```diff
@@ -147,6 +149,33 @@ def _reload_features(self, metadata, features):
         assert callback.features == {"asm": {"enabled": True}}
 
 
+def test_remote_configuration_check_deprecated_var():
+    with override_env(dict(DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS="0.1")):
+        with warnings.catch_warnings(record=True) as capture:
+
+            # Hack: need to pass an argument to configure so that the processors are recreated
+            get_poll_interval_seconds()
+            assert len(capture) == 0
+
+
+def test_remote_configuration_check_deprecated_var_message():
+    with override_env(dict(DD_REMOTECONFIG_POLL_SECONDS="0.1")):
+        with warnings.catch_warnings(record=True) as capture:
+
+            # Hack: need to pass an argument to configure so that the processors are recreated
+            get_poll_interval_seconds()
+            assert len(capture) == 1
+            assert str(capture[0].message).startswith("Using environment")
+
+
+@mock.patch("ddtrace.internal.remoteconfig.worker.deprecate")
+def test_remote_configuration_check_deprecated_override(mock_deprecate):
+    with override_env(dict(DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS="0.1", DD_REMOTECONFIG_POLL_SECONDS="0.5")):
```

Comment:
we might want to consider using `run_python_code_in_subprocess` for these tests, that way we get a clean slate every time and can validate the deprecation behavior without any side effects.

an untested example:

```python
def test_remote_configuration_check_deprecated_override(run_python_code_in_subprocess):

code = """
from from ddtrace.internal.remoteconfig.worker import get_poll_interval_seconds

assert get_poll_interval_seconds() == 0.1
"""

env = os.environ.copy()
env.update(dict(DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS="0.1", DD_REMOTECONFIG_POLL_SECONDS="0.5"))

stdout, stderr, status, _ = run_python_code_in_subprocess(code, env=env)

assert status == 0, stderr
assert stderr == b""
assert b"Using environment ..... " in stdout
```


"ish"

Reply:
Fixed in the last commit ;)